docs/user/quickstart.rst

```diff
@@ -148,7 +148,7 @@ There's also a builtin JSON decoder, in case you're dealing with JSON data::
 
     >>> r = requests.get('https://api.github.com/events')
     >>> r.json()
-    [{u'repository': {u'open_issues': 0, u'url': 'https://github.com/...
+    [{'repository': {'open_issues': 0, u'url': 'https://github.com/...
```

Comment:
Missed one here

Reply:
Thanks! I'll fix it as soon as possible.

Reply:
Fixed!